browser/mocha/fixture.js

```diff
@@ -6,14 +6,22 @@ extendInterfaces('fixture', function (context, teardown) {
   // compatibility with `test-fixture-mocha.js`:
   return context.fixture || function fixture(fixtureId, model) {
 
+    // Find the test-fixture with the provided ID and create it, returning
+    // the results:
+    var fixture = document.getElementById(fixtureId);
+    if (!fixture) {
+      throw new Error('Could not find fixture with ID "' + fixtureId + '"');
+    }
+
     // Automatically register a teardown callback that will restore the
     // test-fixture:
     teardown(function () {
-      document.getElementById(fixtureId).restore();
+      var teardownFixture = document.getElementById(fixtureId);
```

Comment:
This is important because by the time teardown happens that fixture may not be in the DOM for *whatever reason*.